extensions/cdc-postgres/src/main/java/com/hazelcast/jet/cdc/postgres/PostgresCdcSources.java

```diff
@@ -46,12 +46,20 @@ private PostgresCdcSources() {
      * Creates a CDC source that streams change data from a PostgreSQL database
      * to Hazelcast Jet.
      * <p>
-     * If Jet can't reach the database when it attempts to start the source or
-     * if it looses the connection to the database from an already running
-     * source, it throws an exception and terminate the execution of the job.
-     * This behaviour is not ideal, would be much better to try to reconnect,
-     * at least for a certain amount of time. Future versions will address the
-     * problem.
+     * <b>KNOWN ISSUE 1:</b> If Jet can't reach the database when it attempts to
+     * start the source or if it looses the connection to the database from an
+     * already running source, it throws an exception and terminate the
+     * execution of the job. This behaviour is not ideal, would be much better
+     * to try to reconnect, at least for a certain amount of time. Future
+     * versions will address the problem.
+     * <p>
+     * <b>KNOWN ISSUE 2:</b> The Debezium PostgreSQL Connector this source is
+     * based on has data loss issues when synchronizing snapshots with event
+     * streams based on the Postgres WAL (Write-Ahead Log). For details see
+     * <a href="https://issues.redhat.com/browse/DBZ-2288">DBZ-2288</a>.
+     * Currently the first streaming event that should come in right after the
```

Comment:
It seems that more items still can be missed after some circumstances. It also not right for events which came a longer time after snapshot (i.e. if there is not update of DB during snapshotting)